accession-commons-monotonic-generator-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/generators/monotonic/MonotonicAccessionGenerator.java

```diff
@@ -46,13 +46,16 @@
 
     private String applicationInstanceId;
 
+    private long nextBlockInterval;
+
     private ContiguousIdBlockService blockService;
 
     private final BlockManager blockManager;
 
-    public MonotonicAccessionGenerator(long blockSize, String categoryId, String applicationInstanceId,
-                                       ContiguousIdBlockService contiguousIdBlockService) {
+    public MonotonicAccessionGenerator(long blockSize, long nextBlockInterval, String categoryId,
+                          String applicationInstanceId, ContiguousIdBlockService contiguousIdBlockService) {
```

Comment:
Fix indentation, should be aligned with the bracket in the previous line.

Reply:
Fixed